readthedocs/search/documents.py

```diff
@@ -179,7 +179,6 @@ def get_queryset(self):
         queryset = super().get_queryset()
         queryset = (
             queryset
-            .internal()
```

Comment:
We are doing filtering per-version in https://github.com/readthedocs/readthedocs.org/blob/2aa332bee69b9da9a315cc5bced6d1d3fe0340dd/readthedocs/search/utils.py#L28-L31, and we don't trigger the index task for external versions, so this query doesn't change anything.